src/org/labkey/filetransfer/FileTransferController.java

```diff
@@ -53,63 +77,71 @@ public FileTransferController()
         setActionResolver(_actionResolver);
     }
 
-    @RequiresPermission(ReadPermission.class)
-    public class BeginAction extends SimpleViewAction
+    public static ActionURL getComplianceSettingsURL()
     {
-        public ModelAndView getView(Object o, BindException errors) throws Exception
-        {
-            return new FileTransferMetadataView(getViewContext());
-        }
-
-        public NavTree appendNavTrail(NavTree root)
-        {
-            return root;
-        }
+        return new ActionURL(ConfigurationAction.class, ContainerManager.getRoot());
     }
 
+    @CSRF
     @RequiresPermission(AdminOperationsPermission.class)
```

Comment:
See PermissionCheckableAction.java around line 163.

Using @AdminConsoleAction(AdminOperationsPermission.class) = container must be root, user must have AdminReadPermission to GET, user must have AdminOperationsPermission to POST.

Using just @AdminConsoleAction = container must be root, user must have AdminReadPermission to GET, user must have AdminPermission to POST.

And if you want to restrict the GET as well as the POST for an admin console action, use @AdminConsoleAction @RequiresPermission(AdminOperationsPermission.class).